code/renderers/svelte/src/docs/extractArgTypes.ts

```diff
@@ -23,8 +25,16 @@ export const extractArgTypes: ArgTypesExtractor = (
   try {
     // eslint-disable-next-line no-underscore-dangle
     const docgen = component.__docgen;
+
     if (docgen) {
-      return createArgTypes(docgen);
+      const argTypes = createArgTypes(docgen);
+
+      if (!IS_SVELTE_V4) {
+        // NOTE: Filter out event handlers `on:*` as they're deprecated from Svelte v5
+        return Object.fromEntries(Object.entries(argTypes).filter(([name, _]) => !name.startsWith('on:')));
```

Comment:
I also believe it might be possible to optimize performance a bit by using [`.filter()` from `es-toolkit`](https://es-toolkit.slash.page/reference/compat/array/filter.html#filter)

Reply:
Outdated.